Libraries/Lists/VirtualizedList.js

```diff
@@ -503,8 +504,13 @@ class VirtualizedList extends React.PureComponent<OptionalProps, Props, State> {
   }
 
   render() {
+    const flatStyles = flattenStyle(this.props.contentContainerStyle);
+    if (flatStyles) {
+      this._paddingTop = flatStyles.paddingTop || 0;
+      this._paddingBottom = flatStyles.paddingBottom || 0;
```

Comment:
Don't mutate in render - componentDidReceiveProps would be a better place for setting these.